standards/application/relay-reliability.md

```diff
@@ -0,0 +1,150 @@
+---
+title: RELIABILITY-RELAY
+name: Reliability for Relay Protocol
+category: Standards Track
+tags: [reliability, application]
+editor: Kaichao Sun <kaichao@status.im>
+contributors:
+  - Richard Ramos <richard@status.im>
+---
+
+## Abstract
+
+[Relay Protocol](https://github.com/vacp2p/rfc-index/blob/main/waku/standards/core/11/relay.md) in Waku is efficient for routing messages, but there's no guarantee that a message will reach to its destination. For example, the receiver in a chat application may miss messages when network issue happens at either sender or receiver side. In general, a message in Waku network includes 3 status:
+
+- **outgoing**, the message is posted by its creator but no confirmations from other nodes yet
+- **sent**, the message is received by any other node in the network
+- **delivered**, the message is acknowledged by the receiver
+
+Application like Status already uses [MVDS](https://github.com/vacp2p/rfc-index/blob/main/vac/2/mvds.md) for e2e acknowledgement in direct messages and group chat. There is an ongoing [discussion](https://forum.vac.dev/t/end-to-end-reliability-for-scalable-distributed-logs/293) about a more general and bandwidth efficient solution for e2e reliablity.
```

Comment:
Can we be a bit more explicit about what constitutes reliability from application vs Waku POV. In other words, an application defines a payload over Waku and is interested in e2e delivery between application users. Waku is a pub/sub broadcast transport interested in reliably routing a message to all participants in the pub/sub broadcast group.